### PR TITLE
Can now manage seeds with the flash scripts.

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-tools/src/libs/ganache.ts [hunk 1]
```diff
@@ -2,23 +2,36 @@ import * as ganache from "ganache-core";
...
 import memdown from "memdown";
 import { MemDown } from "memdown";
+import { Contracts as compilerOutput } from "@augurproject/artifacts";
 
 import { Account } from "../constants";
+import crypto from "crypto";
+import { EthersProvider } from "@augurproject/ethersjs-provider/build";
+const levelup = require("levelup");
+import * as path from "path";
+import { ContractAddresses } from "@augurproject/artifacts/build";
```
Comment (bthaile) [inline packages/augur-tools/src/libs/ganache.ts]: are we standardizing to use top level references

File: packages/augur-tools/src/libs/ganache.ts [hunk 2]
```diff
@@ -2,23 +2,36 @@ import * as ganache from "ganache-core";
 import { ethers } from "ethers";
 import memdown from "memdown";
 import { MemDown } from "memdown";
+import { Contracts as compilerOutput } from "@augurproject/artifacts";
 
 import { Account } from "../constants";
+import crypto from "crypto";
+import { EthersProvider } from "@augurproject/ethersjs-provider/build";
```
Comment (bthaile) [inline packages/augur-tools/src/libs/ganache.ts]: are we standardizing to use top level references

File: packages/augur-test/src/tests/api/Contracts.test.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { makeProvider, seedPath } from "../../libs";
+import { makeProvider } from "../../libs";
 import { Contracts } from '@augurproject/sdk/build/api/Contracts';
```
Comment (bthaile) [inline packages/augur-test/src/tests/api/Contracts.test.ts]: are we standardizing to use top level references

**Review by bthaile — APPROVED** [review]
lgtm

File: packages/augur-tools/src/libs/ganache.ts (hunk 2 above)
Comment (robert-e-davidson3) [reply packages/augur-tools/src/libs/ganache.ts]: good catch!

**Pull request merged**